tinygrad/optim.py

```diff
@@ -8,23 +8,20 @@ def __init__(self, params):
     self.params = [x for x in params if x.requires_grad == True]
 
   def zero_grad(self):
-    for param in self.params:
-      param.grad = None
+    for param in self.params: param.grad = None
 
 class SGD(Optimizer):
   def __init__(self, params, lr=0.001):
     super().__init__(params)
     self.lr = lr
 
   def step(self):
-    for t in self.params:
-      t -= t.grad * self.lr
+    for t in self.params: t -= t.grad * self.lr
```

Comment:
just a one liner hack idk it works.
 list(map(lambda p :p.assign(p.sub(self.lr*p.grad)) ,self.params))

Reply:
please no `list(map(lambda ...`

Reply:
> please no `list(map(lambda ...`

yeah just giving alternative ... i also wouldnt prefer oneliners..